fe/fe-core/src/main/java/org/apache/doris/catalog/HMSResource.java

```diff
@@ -0,0 +1,84 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.catalog;
+
+import org.apache.doris.common.DdlException;
+import org.apache.doris.common.proc.BaseProcResult;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * HMS resource
+ * <p>
+ * Syntax:
+ * CREATE RESOURCE "hive"
+ * PROPERTIES
+ * (
+ * "type" = "hms",
+ * "hive.metastore.uris" = "thrift://172.21.0.44:7004"
+ * );
+ */
+public class HMSResource extends Resource {
+    // required
+    public static final String HIVE_METASTORE_URIS = "hive.metastore.uris";
```

Comment:
OK, I'm going to move the function of analyzing `hdfs-site.xml` and `hdfs-site.xml` into `HMSResource` when using `Resource` to `CreateCatalogStmt`. Now it is just a framework. If I move the function here, but the previous function cannot be deleted from `CreateCatalogStmt`, it will appear that the code is redundant.